### PR TITLE
Fix CORS errors with error response codes

### PR DESCRIPTION
**Pull request by loneil**
NGINX wasn't returning response codes for error type responses because of CORS, so add the 'always' keyword which I think is the way to overcome that

http://nginx.org/en/docs/http/ngx_http_headers_module.html
```
Adds the specified field to a response header provided that the response code equals 200, 201 (1.3.10), 204, 206, 301, 302, 303, 304, 307 (1.1.16, 1.0.13), or 308 (1.13.0). Parameter value can contain variables.

There could be several add_header directives. These directives are inherited from the previous configuration level if and only if there are no add_header directives defined on the current level.

If the always parameter is specified (1.7.5), the header field will be added regardless of the response code.

```

Reference:
https://serverfault.com/questions/393532/allowing-cross-origin-requests-cors-on-nginx-for-404-responses

### CODE REVIEW

**Review by popkinj — APPROVED** [review]
Make it so Number 1!

File: services/tenant-ui/frontend/src/components/LoginForm.vue [hunk 1]
```diff
@@ -100,26 +100,28 @@ const handleSubmit = async (isFormValid: boolean) => {
...
+        ]);
+      } catch (err) {
+        console.error(err);
+      }
     } catch (err) {
       console.error(err);
+      toast.error(`Failure getting tenant: ${err}`);
+    } finally {
+      submitted.value = false;
     }
```
Comment (loneil) [inline services/tenant-ui/frontend/src/components/LoginForm.vue:124]: Cleaning up unrelated double error message display
Will probably adjust this later now that it knows about 404s as well

**Comment by popkinj** [comment]
If this works as intended I'll revert the workaround I did in the store.

**Comment by popkinj** [comment]
If this works as intended I'll revert the workaround I did in the store.

**Pull request merged**